runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -2228,6 +2228,17 @@ TR_J9ServerVM::targetMethodFromMethodHandle(TR::Compilation* comp, TR::KnownObje
    return NULL;
    }
 
+bool
+TR_J9ServerVM::isInvokeCacheEntryAnArray(uintptr_t *invokeCacheArray)
+   {
+   JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
+   stream->write(
+      JITServer::MessageType::VM_getKnotIndexOfInvokeCacheArrayAppendixElement,
+      invokeCacheArray
+      );
+   return std::get<0>(stream->read<bool>());
```

Comment:
The response from the client does not appear to be a bool. Am I missing something?

Reply:
This was supposed to be `VM_isInvokeCacheEntryAnArray`, but while squashing and rebasing I might have ordered one of the commits incorrectly. I have fixed this to use the right message, where the response type is bool. This also addresses your other comments about whether `VM_isInvokeCacheEntryAnArray` message is used anywhere.